src/subscription_manager/managercli.py

```diff
@@ -68,6 +68,8 @@
 
 from subscription_manager.i18n import ungettext, ugettext as _
 
+from syspurpose.files import SyspurposeStore, USER_SYSPURPOSE
```

Comment:
We don't have a hard dependency on syspurpose (unless I'm unaware that we changed our minds?), so we should always be using `subscription_manager.syspurposelib` as an interface into that stuff. The syspurposelib will need a new method called something like `save_role_to_syspurpose_metadata`, similar to the ones for usage & sla.